src/const_game.h

```diff
@@ -0,0 +1,45 @@
+// hunger thresholds and messages
```

Comment:
The integers, at least, could also be `constexpr` (or a typed enum: hunger::minor-- take your pick).